src/js/experiments/signupLinksCta.js

```diff
@@ -0,0 +1,42 @@
+const isFirstStepsPage = window.location.pathname == '/docs/2.0/first-steps/';
+
+function handleGithubDropdownClick(){
+  const dropdown = $('.gh-signup-dropdown')
+  // toggle the popup
+  $('.gh-dropdown-button').click((e) => {
+    dropdown.addClass("show");
+    e.stopPropagation()
+  })
+
+  // clicking on body closes pop if it is open.
+  $('body').click((e) => {
+    dropdown.removeClass("show");
+  })
+}
+
+/**
+ * query all signup buttons by class
+ * add click listener to track to amplitude before navigating to new page
+ * */
+function handleClickedLink() {
+  $(".track-signup-link").each(function() {
+    $(this).click((e) => {
+      window.AnalyticsClient.trackAction("dd_first-steps-signup-cta_test", {
+        button: e.target.innerText
+      });
+    })
+  })
+
+}
+window.OptimizelyClient.getVariationName({
+  experimentKey: 'dd_first-steps-signup-cta_test',
+  groupExperimentName: 'q1_fy23_docs_disco_experiment_group_test',
+  experimentContainer: '.sign-up-and-try-circleci',
+  guestExperiment: true,
```

Comment:
I dont think we are running this experiment on guest users? If we are only running this experiment on signed in users we can omit `guestExperiment` or set it to false

Reply:
Its my mistake, you're totally right! I got confused and then I realized its CTA buttons.... 😅